lisp/tree-sitter-debug.el

```diff
@@ -76,8 +76,11 @@ This only takes effect if `tree-sitter-debug-jump-buttons' is non-nil."
   ;; TODO: Re-render only affected nodes.
   (when-let ((tree tree-sitter-tree))
     (with-current-buffer tree-sitter-debug--tree-buffer
-      (erase-buffer)
-      (tree-sitter-debug--display-node (tsc-root-node tree) 0))))
+      (buffer-disable-undo)
+      (setq buffer-read-only t)
```

Comment:
I think putting this in `tree-sitter-debug--setup` is cleaner.

Reply:
Done! :)